app/javascript/components/explore/NumericCellFacet.jsx

```diff
@@ -0,0 +1,317 @@
+        if (value === null) {
+          // Count number of cells that have no numeric value for this annotation
+          bars[j].count += count
+        }
+      } else if (j < bars.length - 1) {
+        // If not last bar, use exclusive (<) upper-bound to avoid double-count
+        if (bar.start <= value && value < bar.end) {
+          bars[j].count += count
+        }
+      } else {
+        // If last bar, use inclusive (<=) upper-bound to avoid omitting maximum
+        if (bar.start <= value && value <= bar.end) {
+          bars[j].count += count
+        }
+      }
+    }
+  }
+
+  let maxCount = 0
+  for (let i = 0; i < bars.length; i++) {
+    const count = bars[i].count
+    if (count > maxCount) {maxCount = count}
+  }
+
+  return [bars, maxValue, maxCount]
+}
+
+/** SVG histogram showing distribution of numeric annotation values */
+function Histogram({ filters }) {
+  const maxHeight = 20
+
+  const [bars, maxValue, maxCount, minValue] = getHistogramBars(filters)
+
+  const barRectAttrs = []
+  bars.forEach((bar, i) => {
+    const height = maxHeight * (bar.count / maxCount)
+    const width = 11
+    const attrs = {
+      x: (width + 1) * i,
+      y: maxHeight - height + 1,
+      width,
+      height,
+      color: (bar.isNull) ? '#888' : '#3D5A87',
+      bar
+    }
+    barRectAttrs.push(attrs)
+  })
+
+  const lastBar = barRectAttrs.slice(-1)[0]
+  const svgHeight = maxHeight + 2
+  const svgWidth = lastBar.x + lastBar.width
+  return (
+    <>
+      <svg
+        height={svgHeight}
+        width={svgWidth}
+        style={{ borderBottom: '1px solid #AAA  ' }}
+        className="numeric-filter-histogram"
+      >
+        {barRectAttrs.map((attrs, i) => {
+          return (
+            <rect
+              fill={attrs.color}
+              x={attrs.x}
+              y={attrs.y}
+              width={attrs.width}
+              height={attrs.height}
+              key={i}
+            />
+          )
+        })}
+      </svg>
+      <div style={{ position: 'absolute', top: 0 }} key={2}>
+        {barRectAttrs.map((attrs, i) => {
+          const bar = attrs.bar
+          let criteria
+          if (bar.start === null) {
+            criteria = 'N/A'
+          } else {
+            const briefStart = round(bar.start, 2)
+            const briefEnd = round(bar.end, 2)
+            criteria = `${briefStart}&nbsp;-&nbsp;${briefEnd}`
+          }
+          const tooltipContent = `<span>${criteria}:<br/>${bar.count}&nbsp;cells</span>`
+
+          return (
+            <span
+              style={{
+                display: 'inline-block',
+                width: attrs.width + 1,
+                height: maxHeight
+              }}
+              data-toggle="tooltip"
+              data-html={true}
+              data-original-title={tooltipContent}
+              key={i}
+            >
+            </span>
+          )
+        })}
+      </div>
+    </>
+  )
+}
+
+const operators = [
```

Comment:
Did we say during the last demo that we were going to use the mathematical operators for the more basic functions instead of the plain text?  E.g. `>=` for `greater than or equal to`?

Reply:
Thanks for reminding -- done in a44608c.